frontend/src/components/profile/ProfileHeader.tsx

```diff
@@ -22,7 +22,7 @@ export default function ProfileHeader({ user }: ProfileHeaderProps) {
       <Stack direction="column">
         <Stack direction="row" alignItems="center" spacing={1}>
           <Typography variant="subtitle1">{user.name}</Typography>
-          {user.pronoun && (
+          {user.pronoun && user.pronoun.name !== "Prefer not to say" && (
```

Comment:
Perhaps enum would be better:

``` 
enum EPronoun {
  HeHis = 1, 
  SheHer = 2,
  PreferNotToSay = 3
}
```

then, use like so: 
```
user.pronoun && user.pronoun.id !== EPronoun.PreferNotToSay
```

Reply:
Good idea.

Reply:
Using string instead of id, since backend seed file doens't explicity state the id. (It's auto increment)